views/_includes/buttonMixin.pug

```diff
@@ -5,9 +5,12 @@ mixin formButtons(submitButtonText = 'Continue', cancelUrl='/clear')
     .buttons--right
       a.transparent.full-width(href=cancelUrl role='button' draggable='false') #{__('Cancel')}
 
-mixin linkButtons(confirmURL)
+mixin linkButtons(confirmURL, continueButton)
   .buttons-row
-    a(href=confirmURL role='button' draggable='false') #{__('Confirm')}
+    if continueButton
+      a(href=confirmURL role='button' draggable='false') #{__('Continue')}
+    else
+      a(href=confirmURL role='button' draggable='false') #{__('Confirm')}
```

Comment:
The only difference between these two seems to be the button text. 

Let's just pass in the button text, but default to 'Confirm', like we do here: https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/views/_includes/buttonMixin.pug#L1